src/status_im/ui/components/connectivity/styles.cljs

```diff
@@ -3,21 +3,23 @@
   (:require [status-im.ui.components.colors :as colors]
             [status-im.utils.platform :as platform]))
 
-(defnstyle text-wrapper [top opacity window-width pending?]
+(defnstyle text-wrapper [{:keys [top window-width pending? modal?]}]
   (cond->
-   {:opacity          opacity
+   {:opacity          1.0
```

Comment:
just wondering why 1`.0` ?

Reply:
https://github.com/status-im/status-react/pull/6068/files#diff-df3f46463da5ee34e009067a466d7039L24 it was passed as a parameter before, idk